scripts/amm/stablepool/README.md

```diff
@@ -0,0 +1,16 @@
+## Stablepool deployment
+
+1. Export wallet to a JSON format and save it as `scripts/amm/stablepool/deployerWallet.json`.
```

Comment:
Why did you decide to do it this way? Everywhere else we supply the seed.

Reply:
It seemed more convenient this way but on second thought it may be more hassle than just using the seed phrase :+1: